addons/event_sale/wizard/event_configurator.py

```diff
@@ -22,3 +24,20 @@ def check_event_id(self):
                     _('Invalid ticket choice "%(ticket_name)s" for event "%(event_name)s".'))
         if error_messages:
             raise ValidationError('\n'.join(error_messages))
+
+    @api.depends('product_id')
+    def _compute_has_available_tickets(self):
+        for configurator in self:
+            configurator.has_available_tickets = bool(
+                self.env['event.event'].search_count([
+                    ('event_ticket_ids.product_id', '=', configurator.product_id.id),
+                    ('date_end', '>=', fields.Date.today())]))
```

Comment:
As tickets cannot exist without their matching event (they're cascade deleted), wouldn't it be more efficient to search on the event.ticket model directly?

Also, arguable, you could use a read_group with a count, grouped by product_id, to make it multi-enabled.
(Even if it is not THAT important for wizards, it's good to make everything batch-enabled when possible).